cli/args.go

```diff
@@ -87,6 +87,11 @@ func parseTerragruntOptionsFromArgs(args []string, writer, errWriter io.Writer)
 		return nil, err
 	}
 
+	ExcludeDirs, err := parseStringArg(args, OPT_TERRAGRUNT_EXCLUDE_DIR, os.Getenv("TERRAGRUNT_EXCLUDE_DIR"))
```

Comment:
Doesn't `parseStringArg` parse a string rather than a slice?

Reply:
Correct. Sorry my local git was messed up and I did a partial commit, should be fixed now.